users/documentation/index.md

```diff
@@ -22,7 +22,12 @@ itself, and a reference guide to the built-in special functions and probability
 functions.  Stan's modeling language is portable across all interfaces.
 
 * <p>
-  <a href="https://github.com/stan-dev/stan/releases/download/v2.17.0/stan-reference-2.17.0.pdf"><i>Modeling Language User's Guide and Reference Manual</i>, Version 2.17.0</a>
+  <a href="https://github.com/stan-dev/stan/releases/download/v2.18.0/users-guide-2.18.0.pdf"><i>Modeling Language User's Guide</i>, Version 2.18.0</a>
```

Comment:
There should be a reference manual pdf here, too.  There's also the reference manual HTML which needs to be hosted somehwere in order for search to work.